guides/v2.1/install-gde/prereq/prereq_updater.md

```diff
@@ -1,18 +1,163 @@
 ---
 group: install_pre
-subgroup: Prerequisites
-title: Set up the updater (obsolete)
-menu_title: Set up the updater (obsolete)
-menu_order: 1000
-menu_node:
-version: 2.1
+title: Set up the updater
 github_link: install-gde/prereq/prereq_updater.md
 functional_areas:
   - Install
   - System
   - Setup
 ---
 
-This topic is obsolete. Magento no longer has a separate updater application.
+The updater synchronizes with Magento Connect for updating components and upgrading the Magento software.
```

Comment:
`Magento Connect` is obsolete term.
I'd describe it in a more generic way - the Updater uses Composer to manage Magento packages. It may get packages from https://packagist.org/ , not only from Magento Marketplace.